### PR TITLE
MOB-1637 - Hot feature suggestions

### PR DESCRIPTION
**Pull request by Oleg-Pecheneg**

### CODE REVIEW

File: unstoppable-ios-app/domains-manager-ios-preview/Modules/PreviewDomainsCollection/PreviewDomainsCollectionViewController.swift [hunk 1]
```diff
@@ -9,6 +9,15 @@ import SwiftUI
 
 @available(iOS 17, *)
 #Preview {
+    HotFeatureSuggestionsStorage.clearAll()
+    
+    
+//    let router = DomainsCollectionRouter()
```
Comment (rommex) [inline unstoppable-ios-app/domains-manager-ios-preview/Modules/PreviewDomainsCollection/PreviewDomainsCollectionViewController.swift:15]: Dead?

File: unstoppable-ios-app/domains-manager-ios-preview/Modules/PreviewDomainsCollection/PreviewDomainsCollectionViewController.swift [hunk 2]
```diff
@@ -36,17 +46,21 @@ final class PreviewDomainsCollectionViewPresenter {
 // MARK: - DomainsCollectionPresenterProtocol
 extension PreviewDomainsCollectionViewPresenter: DomainsCollectionPresenterProtocol {
     func viewDidLoad() {
-        UserDefaults.didShowSwipeDomainCardTutorial = false
+//        UserDefaults.didShowSwipeDomainCardTutorial = false
```
Comment (rommex) [inline unstoppable-ios-app/domains-manager-ios-preview/Modules/PreviewDomainsCollection/PreviewDomainsCollectionViewController.swift:49]: Dead?

**Review by rommex — APPROVED** [review]

File: unstoppable-ios-app/domains-manager-ios-preview/Modules/PreviewDomainsCollection/PreviewDomainsCollectionViewController.swift (hunk 1 above)
Comment (Oleg-Pecheneg) [reply unstoppable-ios-app/domains-manager-ios-preview/Modules/PreviewDomainsCollection/PreviewDomainsCollectionViewController.swift:15]: Preview code

File: unstoppable-ios-app/domains-manager-ios-preview/Modules/PreviewDomainsCollection/PreviewDomainsCollectionViewController.swift (hunk 2 above)
Comment (Oleg-Pecheneg) [reply unstoppable-ios-app/domains-manager-ios-preview/Modules/PreviewDomainsCollection/PreviewDomainsCollectionViewController.swift:49]: Preview code

**Pull request merged**